documentation/en/README.md

```diff
@@ -0,0 +1,39 @@
+# GeoWebCache Documentation
+
+## Building
+
+An ant build script is required (note the python environment described below is required).
+
+```
+ant docs
+```
+
+For feedback while editing:
+```
+ant site
+```
+
+## Setup Python Virtual Environment
+
+Using `virtualenv`` (macos example):
+
+```
+brew install virtualenv
+virtualenv venv
+```
+
+To activate python:
+
+```
+source venv/bin/activate
+```
+
+To configure this environment with ``sphinx-build`` (and any other ``requirements.txt``):
+```
+pip install -r requirements.txt
+```
+
+To confirm installation:
+```
+sphinx-build --version
+```
```

Comment:
These are inconsistent with the [GeoServer sphinx install instructions](https://docs.geoserver.org/latest/en/docguide/install.html), which, notably, don't use a venv to install sphinx. 

I think the appropriate change is probably to update the GeoServer instructions, but at the moment, if anyone has set up an environment to build GeoServer docs already, these instructions are going to result in an extra venv.

Reply:
Yes I think so; if I understand correctly the requirements.txt is useful with pip (no venv required)

Reply:
Fair enough, this technically would allow us to specify the version of sphinx supported/tested by *this* build.

I am going to reword so that the venv is optional.